lib/tailwind_formatter.ex

```diff
@@ -118,10 +101,16 @@ defmodule TailwindFormatter do
   end
 
   defp sort(classes) when is_binary(classes) do
-    classes
-    |> sort_variant_chains()
-    |> sort()
-    |> Enum.join(" ")
+    leading_space = if String.starts_with?(classes, " "), do: " "
+    trailing_space = if String.ends_with?(classes, " "), do: " "
+
+    classes =
+      classes
+      |> sort_variant_chains()
+      |> sort()
+      |> Enum.join(" ")
+
+    Enum.join([leading_space, classes, trailing_space])
```

Comment:
I decided to preserve a single leading or a single trailing space. This way, the user ends up having control of where to put a space when concatenating.